src/index.js

```diff
@@ -143,6 +142,9 @@ export class IncogniaAPI {
     return axios({
       method: Method.POST,
       url: this.apiEndpoints.TOKEN,
+      data: {
```

Comment:
esse data vai como body do urlencoded mesmo né? perguntando pq no java eu tive que colocar uma string

Reply:
Vish, aparentemente não. Tá assim:

```
POST /v1/token HTTP/1.1
Accept: application/json, text/plain, */*
Content-Type: application/x-www-form-urlencoded
User-Agent: axios/0.21.1
Content-Length: 35
Host: localhost:9000
Authorization: Basic xalala
Connection: close

{"grant_type":"client_credentials"}
```

Precisa ser assim, né? :point_down: 
```
POST / HTTP/1.1
Host: localhost:9000
User-Agent: curl/7.68.0
Accept: */*
content-type: application/x-www-form-urlencoded
Content-Length: 116

grant_type=client_credentials
```

Pior que eu fiz como sugerem com Node/axios aqui no [post do Auth0](https://auth0.com/docs/flows/call-your-api-using-the-client-credentials-flow#parameters) e ficou do mesmo jeito. :roll_eyes:

Reply:
@julianalucena acho que tu precisa enviar como params

```suggestion
      params: {
```

Reply:
Com a sugestão de Ottony, fica assim:

```
POST /v1/token?grant_type=client_credentials HTTP/1.1
Accept: application/json, text/plain, */*
Content-Type: application/x-www-form-urlencoded
User-Agent: axios/0.21.1
Host: localhost:9000
Authorization: Basic xalala
Connection: close
Content-Length: 0
```

Reply:
Ficou assim, amigos:
```
➜  incognia-api-node-wrapper git:(fix/token-related-fixes) ✗ nc -l 9000
POST /v1/token HTTP/1.1
Accept: application/json, text/plain, */*
Content-Type: application/x-www-form-urlencoded
User-Agent: axios/0.21.1
Content-Length: 29
Host: localhost:9000
Authorization: Basic xalala
Connection: close

grant_type=client_credentials
```

https://github.com/inloco/incognia-api-node-wrapper/pull/11/commits/ee789f743994eb36a24c3098588b98cf35f6f94f